test/extended/prometheus/prometheus.go

```diff
@@ -184,6 +184,17 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			}
 			runQueries(tests, oc, ns, execPod.Name, url, bearerToken)
 		})
+		g.It("shouldn't have failing rules evaluation", func() {
+			oc.SetupProject()
+			ns := oc.Namespace()
+			execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
+			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
+
+			tests := map[string]bool{
+				`prometheus_rule_evaluation_failures_total >= 1`: false,
```

Comment:
I thought we had two rule failures, should this not have failed 🤔

Reply:
we can take a look in prometheus dump and see what happened :)

Reply:
I've used promecieus to check a dump and test is ok, during whole test run there was almost 15k rule evaluations and 0 failures according to prometheus metrics.

Reply:
Thanks for following up! Must have been a flake then on the cluster that I seen or something was fixed in between. 🤷‍♀